ui/app/src/components/UserManagement/utils.ts

```diff
@@ -30,7 +30,11 @@ export const minLengthMap = {
 };
 
 export const isInvalidEmail = (email: string) => {
-  const emailRegex = /^([\w\d._\-#])+@([\w\d._\-#]+[.][\w\d._\-#]+)+$/g;
+  // From https://emailregex.com/, according to the RFC 5322 standard
+  const emailRegex =
+    // Linter shows a couple of warnings, but the regex is correct
+    // eslint-disable-next-line
+    /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/gm;
```

Comment:
What's the need for the global and multiline flags?

Reply:
They're actually not needed. Removing